### DISCUSSION TITLE
Questions regarding js course

### DISCUSSION THREAD

**Discussion by Ronak3108** [community/community#38427] [Other Feature Feedback, Questions, & Ideas]
I have 3 questions :-
1. Would we get a video on math Objects?
2. How do I explore more about other node packages?
3. Would we get another course of backend js or node.js?
 
I expect reply from you side.
Thank you.

(I don't know clearly how to start a discussion on GitHub. Sorry If I made any mistake in starting a discussion)

**Comment by airtower-luna** [comment]
You might want to add which course you mean. This is a place mostly for discussion around Git and GitHub, so it's not obvious exactly what you're talking about. :wink:
